cmd/agent/app/processors/thrift_processor.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/uber/jaeger-lib/metrics"
 	"go.uber.org/zap"
 
-	"github.com/jaegertracing/jaeger/cmd/agent/app/customtransports"
+	customtransport "github.com/jaegertracing/jaeger/cmd/agent/app/customtransports"
```

Comment:
This line is needed otherwise line 66 will not compile. I am not sure how it was compiling before. I may be missing something.

Reply:
Suggestion: Rename the directory the the package is in or the package itself to match the directory name. Right now the package is named `customtransport`, which the directory is `customertransports`.

Reply:
Yeah, that would be the right approach, otherwise it breaks a common naming practice with no apparent good reason.

However, I am reluctant to make this change. If any other external code (not in this repo) imports this package renaming it will break that external code. I'd rather let this repo's maintainers decide if that's what they want to do.

I am reverting this change since apparently it compiles fine the way it was, not sure why I thought it doesn't.